bindings/node.js/test/kzg.test.ts

```diff
@@ -19,17 +19,17 @@ import {
   BYTES_PER_PROOF,
   BYTES_PER_FIELD_ELEMENT,
   transformTrustedSetupJSON,
-} from "./kzg";
+} from "../lib/kzg";
 
-const setupFileName = "testing_trusted_setups.json";
-
-const SETUP_FILE_PATH = existsSync(setupFileName)
-  ? setupFileName
-  : `../../src/${setupFileName}`;
```

Comment:
This file is committed to the repo.  I removed this ternary because it should always exist